sale_margin_delivered/migrations/16.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if env["ir.module.module"].search(
+        [("name", "=", "sale_margin_delivered"), ("state", "=", "installed")]
+    ):
+        sale_margin_delivered_security = env["ir.module.module"].search(
+            [
+                ("name", "=", "sale_margin_delivered_security"),
+                ("state", "=", "uninstalled"),
+            ]
+        )
+        if sale_margin_delivered_security:
+            sale_margin_delivered_security.button_immediate_install()
```

Comment:
Don't do this, but just change the state to `to install`.

Reply:
corrected :)